src/NServiceBus.Core/EndpointConfiguration.cs

```diff
@@ -178,6 +178,16 @@ public void OverridePublicReturnAddress(string address)
         }
 
         /// <summary>
+        /// Overrides the local address.
+        /// </summary>
+        /// <param name="transportAddress">The transportAddress.</param>
+        public void OverrideLocalAddress(string transportAddress)
```

Comment:
shouldn't that just be an extension method which contains the code from line 231-234?